openassessment/xblock/test/test_openassessment.py

```diff
@@ -59,6 +59,33 @@ def test_load_student_view(self, xblock):
         self.assertIsNotNone(grade_response)
         self.assertIn("step--grade", grade_response.body)
 
+    @scenario('data/staff_grade_scenario.xml')
+    def test_load_grade_available_responses_view(self, xblock):
```

Comment:
As above, I could see naming this `test_staff_assessment_view`. I'm not super picky which we end up with, as long it's consistent 😀